dist/streams.js

```diff
@@ -1,36 +1,39 @@
 import EventEmitter from './events.js';
 
-const encoder = new TextEncoder();
-const decoder = new TextDecoder();
-const metadataPattern = /^mc:-?\d+;$/;
-const finisherByte = 59; // ;
+const numToBytesInt32 = (num) => {
+  const numBytes = new ArrayBuffer(4);
+  const numView = new DataView(numBytes);
+  numView.setInt32(0, num);
+  return numBytes;
+};
+
+const numFromBytesInt32 = (buffer) => {
+  const numView = new DataView(buffer);
+  return numView.getInt32(buffer.length);
```

Comment:
`streamId` starts from 0 offset:
```suggestion
  return numView.getInt32(0);
```